jerry-core/jerry.c

```diff
@@ -635,6 +635,24 @@ jerry_api_release_object (jerry_api_object_t *object_p) /**< pointer acquired th
 } /* jerry_api_release_object */
 
 /**
+ * Acquire specified Jerry API value
+ */
+jerry_api_value_t *
+jerry_api_acquire_value (jerry_api_value_t *value_p) /**< API value */
+{
+  jerry_assert_api_available ();
+
+  if (value_p->type == JERRY_API_DATA_TYPE_STRING)
+  {
+    jerry_api_acquire_string (value_p->u.v_string);
+  }
+  else if (value_p->type == JERRY_API_DATA_TYPE_OBJECT)
+  {
+    jerry_api_acquire_object (value_p->u.v_object);
+  }
```

Comment:
@akiss77 In my opinion, this is an implementation detail. Perhaps in the future, new data types have acquire/release semantics as well, in which case `jerry_api_acquire_value` and `jerry_api_release_value` should do the right thing. Note that `jerry_api_release_value` does not have an else clause either.